scripts/install_evmone.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env sh
+set -e
+
+TEMPDIR="src"
+cd /
+mkdir -p $TEMPDIR
+cd $TEMPDIR
+git clone https://github.com/chfast/evmone.git
+(
+  cd evmone
+  git submodule update --init --recursive
```

Comment:
Is there a way to combine with with the `clone` step?

Reply:
done.